dom/telephony/TelephonyDialCallback.cpp

```diff
@@ -21,7 +21,6 @@ TelephonyDialCallback::TelephonyDialCallback(nsPIDOMWindow* aWindow,
                                              Promise* aPromise)
   : TelephonyCallback(aPromise), mWindow(aWindow), mTelephony(aTelephony)
 {
-  MOZ_ASSERT(mTelephony);
```

Comment:
Remove this verification, because USSDSession will create TelephonyDialCallback with an empty mTelephony pointer.
This is not dangerous because methods that required a mTelephony pointer will never be called with such a callback.
TODO: find a way to get a mTelephony pointer from USSDSession maybe